lib/entities/npcs/animals/piggy.dart

```diff
@@ -125,6 +125,8 @@ class Piggy extends NPC {
 		SkillManager.learn(SKILL, email);
 		nibbleCounts[email] = (nibbleCounts[email] ?? 0) + 1;
 
+		QuestEndpoint.questLogCache[email].offerQuest('Q11');
```

Comment:
Can you add a comment here and say what quest this will be starting? The way we have the quest files organized, it is hard to glance at what Q11 associates with.